verilog/tools/ls/verilog_ls.cc

```diff
@@ -13,208 +13,50 @@
 // limitations under the License.
 //
 
+#include <functional>
 #include <iomanip>  // Only needed for json debugging right now
 #include <iostream>
 
-#include "common/lsp/json-rpc-dispatcher.h"
-#include "common/lsp/lsp-protocol.h"
-#include "common/lsp/message-stream-splitter.h"
+// Input and output is stdin and stdout
+static constexpr int in_fd = 0;  // STDIN_FILENO
```

Comment:
I'd not put some variable declarations in the middle between include files.

In fact, this constant should probably only be defined within `main()`.

Reply:
I moved declaration to `main()`